apiv2/apiv2_utils.py

```diff
@@ -74,7 +75,8 @@ def hash(self):
 def get_view_description(cls, html=False):
     description = ''
     if getattr(cls, 'get_description', None):
-        cache_key = cls.get_view_name()
+        cache_key = create_hash(cls.get_view_name(), add_secret=False, limit=32)
+        print cache_key
```

Comment:
no print needed, but otherwise it's fine

Reply:
oups hehe, shame on me